cmd/skaffold/app/cmd/flags.go

```diff
@@ -606,6 +606,22 @@ var flagRegistry = []Flag{
 		FlagAddMethod: "BoolVar",
 		DefinedOn:     []string{"deploy"},
 	},
+	{
+		Name:          "transformAllowFile",
+		Usage:         "Path to JSON file specifying the allow list of yaml objects for skaffold to NOT transform with 'image' and 'label' field replacements. NOTE: this list is additive to skaffold's default allowlist",
```

Comment:
what is the precedence between `alloy` and `deny` for the same yaml object? We could mention that here in the flag descriptions.

Reply:
denylist has priority over allowlist.  I've added a note in the flag description